pyiron_atomistics/lammps/base.py

```diff
@@ -427,10 +428,18 @@ def save_output(
             data_dict=output_dict,
             group_name="output",
         )
-        final_structure = self.structure.copy()
-        final_structure.indices = hdf_dict["output/generic/indices"][-1]
-        final_structure.positions = hdf_dict["output/generic/positions"][-1]
-        final_structure.cell = hdf_dict["output/generic/cells"][-1]
+        try:
```

Comment:
Rather than using a `try except` statement, can we compare the length of the initial structure and the indices ?